tests/e2e/actions.go

```diff
@@ -250,6 +258,125 @@ func (tr Chain) submitTextProposal(
+		ValidatorsPowerCap: action.ValidatorsPowerCap,
+		ValidatorSetCap:    action.ValidatorSetCap,
+		Allowlist:          action.Allowlist,
+		Denylist:           action.Denylist,
+		MinStake:           action.MinStake,
+		AllowInactiveVals:  action.AllowInactiveVals,
+	}
+
+	consumerID := tr.testConfig.chainConfigs[action.ConsumerChain].ConsumerId
+	update := types.MsgUpdateConsumer{
+		ConsumerId:               string(consumerID),
+		NewOwnerAddress:          action.NewOwner,
+		InitializationParameters: &initParams,
+		PowerShapingParameters:   &powerShapingParams,
+	}
+	tr.UpdateConsumer(action.Chain, action.From, update)
+}
+
+type InitializeConsumerChainAction struct {
+	Chain               ChainID
+	From                ValidatorID
+	ConsumerChain       ChainID
+	SpawnTime           uint
+	InitialHeight       clienttypes.Height
+	DistributionChannel string
+	TopN                uint32
+	ValidatorsPowerCap  uint32
+	ValidatorSetCap     uint32
+	Allowlist           []string
+	Denylist            []string
+	MinStake            uint64
+	AllowInactiveVals   bool
+}
+
+// initializeConsumerChain creates and initializes a consumer chain
+func (tr Chain) initializeConsumerChain(action InitializeConsumerChainAction, verbose bool) {
+
+	spawn_time := tr.testConfig.containerConfig.Now.Add(time.Duration(action.SpawnTime) * time.Millisecond)
+	params := ccvtypes.DefaultParams()
+	initParams := types.ConsumerInitializationParameters{
+		InitialHeight: action.InitialHeight,
+		GenesisHash:   []byte("gen_hash"),
+		BinaryHash:    []byte("bin_hash"),
+		SpawnTime:     spawn_time,
+
+		UnbondingPeriod:                   params.UnbondingPeriod,
+		CcvTimeoutPeriod:                  params.CcvTimeoutPeriod,
+		TransferTimeoutPeriod:             params.TransferTimeoutPeriod,
+		ConsumerRedistributionFraction:    params.ConsumerRedistributionFraction,
+		BlocksPerDistributionTransmission: params.BlocksPerDistributionTransmission,
+		HistoricalEntries:                 params.HistoricalEntries,
+		DistributionTransmissionChannel:   action.DistributionChannel,
+	}
+
+	powerShapingParams := types.PowerShapingParameters{
+		Top_N:              action.TopN,
+		ValidatorsPowerCap: action.ValidatorsPowerCap,
+		ValidatorSetCap:    action.ValidatorSetCap,
+		Allowlist:          action.Allowlist,
+		Denylist:           action.Denylist,
+		MinStake:           action.MinStake,
+		AllowInactiveVals:  action.AllowInactiveVals,
+	}
+
+	metadata := types.ConsumerMetadata{
+		Name:        "chain name of " + string(action.Chain),
+		Description: "no description",
+		Metadata:    "no metadata",
+	}
+
+	// create consumer to get a consumer-id
+	consumerID := tr.CreateConsumer(action.Chain, action.ConsumerChain, action.From, metadata, &initParams, nil)
+
+	// Update consumer
+	update := types.MsgUpdateConsumer{
+		ConsumerId:             consumerID,
+		PowerShapingParameters: &powerShapingParams,
+	}
+	tr.UpdateConsumer(action.Chain, action.From, update)
```

Comment:
Why do we need to call `UpdateConsumer` here? Couldn't we pass the desired power-shaping parameters in the `CreateConsumer` call?